README.md

```diff
@@ -1,2 +1,23 @@
 # skycoin-services
+
 skycon-services
+
+## dmsg daemon
+
+This daemon will create `.dmsg-uuid` file, which contains your unique UUID.  
+It also log all the actions and results to the `dmsg-uuid.log` file.  
+These files will be created in the same directory, where **dmsgd** are running.
+
+- **port** - to start HTTP server on
+- **disc** - dmsg discovery server
+- **sk** - dmsg server secret key
+- **log-dir** - directory to store logs and UUID data
+
+### Building from source
+
+1. `go build -o dmsgd ./dmsg`
+2. `./dmsgd`
+
+### Running in Docker
+
+1. `docker build -t dmsgd . && docker run -d -v /$PWD/docker_container:/dmsgd-data dmsgd --port=80 --disc="http://dmsg.discovery.skywire.skycoin.com" --sk=***`
```

Comment:
Configuring the container with command flag/arguments gets a bit annoying. It may be preferable to mount a config similar to https://github.com/skycoin/skywire/pull/794

I also can build the docker container and it seems to run but does not return any logs. Not sure if that is intended or if that indicates the container not working. Running `docker ps` seems to indicate we are not actually exposing any port of the container. 

Most people also need some instruction they can copy paste, so I would suggest picking a port other than :80 as it is commonly used by other applications already.